src/Tickets/Admin/All_Tickets/List_Table.php

```diff
@@ -541,14 +690,22 @@ public function get_query_args(): array {
 	 * @since TBD
 	 */
 	public function prepare_items() {
-		global $wpdb;
 
-		$args               = $this->get_query_args();
-		$tickets_repository = tribe_tickets()->by_args( $args );
-		$total_items        = $tickets_repository->found();
-		$items              = $tickets_repository->all();
+		add_filter( 'posts_clauses', [ $this, 'filter_query_clauses' ], 10, 2 );
+
+		$args        = $this->get_query_args();
+		$this->query = new WP_Query( $args );
+		$total_items = $this->query->found_posts;
+		$items       = $this->query->posts;
```

Comment:
I don't want to cause any big changes here, but this is starting to look a lot more like the `WP_Posts_List_Table` class. Could you please have a look into extending this one instead of `WP_List_Table` ? Maybe worth it to have a chat about it.

If the change is not worth it, just let it as is. I believe though, that we could take advantage of functionalities being already offered by the Post List Table class.

Reply:
I thought about this, but because we're displaying multiple `post_types`, I ultimately went against it. `WP_Posts_List_Table` seemed to be specifically designed for a single `post_type`.

Reply:
That's a good point. Yea if we need to have different post types in the same list then `WP_List_Table` is a better option